scripts/content.js

```diff
@@ -150,6 +150,10 @@ function dohl () {
      * Enclose <pre></pre> blocks content in <code></code> blocks
      */
     for (const pre of preList) {
+        /* Replace all <br> tags with newline characters */
+        const codeBlock = pre.querySelector('code');
+        codeBlock.innerHTML = codeBlock.innerHTML.replace(/<br[ /]*>/g, '\n');
```

Comment:
I tested on my side - This line fails and breaks the script if the `<pre>` block does not contain a `<code>`, which is typically the case when you display a text file with Firefox. We should only run the replace if the query selector found something.

Reply:
My bad, I just made another commit that fix that 
173f72ce9aca7138a643dc358f1e8f7ddd342d68